arangograph/notebooks.md

```diff
@@ -59,3 +69,83 @@ at anytime. The notebook's phase is updated accordingly.
 
 1. Select the notebook that you want to remove from the **Notebooks** tab.
 2. Click the **Delete notebook** button.
+
+## Getting Started notebook
+
+To get a better understanding of how to interact with your ArangoDB database
+cluster, use the ArangoGraph Getting Started template.
+The ArangoGraph Notebook automatically connects to the ArangoDB service
+endpoint, so you can immediately start interacting with it.
+
+1. Log in into the notebook you have created.
+2. Open the file browser and navigate to the `jovyan` directory.
```

Comment:
What's this step about? Don't see it on the UI. The Getting started notebook is just displayed in the file tree.

Reply:
At the time I opened this PR, the Getting Started template was "hidden" under the jovyan directory, not really in plain sight. Hence, this step. 
Now, looks like the template is displayed in the file tree right after opening the notebook. I removed the step.